.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: tests
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '42 5 * * *'
```

Comment:
Why every morning? It runs on a push and on a PR, that should be sufficient. We have no checks against min/max composer deps, so not needed IMHO.